<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Moradores</class>
 <widget class="QWidget" name="frm_Moradores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background: #0033A0;
}</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1061</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 26px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>2. Moradores</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Nome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Nome completo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 26px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Morador 1 (Responsável):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_idade">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Idade:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Telefone(contato):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Contato">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
   <property name="inputMask">
    <string>(00) 0 0000-0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Gênero:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f0f0f0;
    border: 1px solid #888;
    border-radius: 6px;
    padding: 5px 10px;
    font-size: 14px;
	color: #000000;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #aaa;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    selection-background-color: #4CAF50;
    selection-color: #9C27B0;
    border: 1px solid #aaa;
    padding: 4px;
    font-size: 13px;
}
</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feminino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prefiro não responder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Outro</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Gênero:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Nome1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_idade1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Idade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Telefone(contato):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>650</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f0f0f0;
    border: 1px solid #888;
    border-radius: 6px;
    padding: 5px 10px;
    font-size: 14px;
	color: #000000;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #aaa;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    selection-background-color: #4CAF50;
    selection-color: #9C27B0;
    border: 1px solid #aaa;
    padding: 4px;
    font-size: 13px;
}
</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feminino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prefiro não responder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Outro</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txt_email1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Nome completo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 26px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Morador 2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_contato1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>580</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
   <property name="inputMask">
    <string>(00) 0 0000-0000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cadastrarM">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #E0E7FF;
    color: #0033A0;
    font-size: 16px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, Tahoma, Geneva, Verdana, sans-serif;
    padding: 10px 22px;
    border: none;
    border-radius: 10px;
    box-shadow: 2px 2px 6px rgba(0, 0, 0, 0.3);
}
QPushButton:hover {
    background-color: #C7D2FE;
}
QPushButton:pressed {
    background-color: #32CD32;
}</string>
   </property>
   <property name="text">
    <string>Cadastrar mais moradores</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Gênero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_13">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_14">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Idade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_15">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Telefone(contato):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_contato2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
   <property name="inputMask">
    <string>(00) 0 0000-0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_idade2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 26px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Morador 3:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f0f0f0;
    border: 1px solid #888;
    border-radius: 6px;
    padding: 5px 10px;
    font-size: 14px;
	color: #000000;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #aaa;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    selection-background-color: #4CAF50;
    selection-color: #9C27B0;
    border: 1px solid #aaa;
    padding: 4px;
    font-size: 13px;
}
</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feminino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prefiro não responder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Outro</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txt_Nome_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;  
    color: #1E1E1E;             
    font-size: 16px;
    padding: 6px 10px;
    border: 1px solid #CCCCCC;
    border-radius: 6px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_coordernadas_16">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF;
    font-size: 18px;
    font-weight: 500;
    background: transparent;
    qproperty-alignment: 'AlignCenter';
}
</string>
   </property>
   <property name="text">
    <string>Nome completo:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>20</width>
     <height>641</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>1061</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Finalizar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>580</y>
     <width>350</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #E0E7FF;
    color: #0033A0;
    font-size: 16px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, Tahoma, Geneva, Verdana, sans-serif;
    padding: 10px 22px;
    border: none;
    border-radius: 10px;
    box-shadow: 2px 2px 6px rgba(0, 0, 0, 0.3);
}
QPushButton:hover {
    background-color: #C7D2FE;
}
QPushButton:pressed {
    background-color: #32CD32;
}</string>
   </property>
   <property name="text">
    <string>Finalizar cadastro de moradores</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>60</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
